src/Components/Facility/BedDeleteDialog.tsx

```diff
@@ -45,10 +34,10 @@ const BedDeleteDialog = (
             {name}
           </p>
           <p className="inline">?</p>
-        </DialogContentText>
-      </DialogContent>
-      <DialogActions>
-        <div className="flex flex-col md:flex-row gap-2 w-full md:justify-end">
+        </div>
+      </div>
+      <div>
+        <div className="flex flex-col md:flex-row gap-2 w-full md:justify-end mt-6">
           <button
```

Comment:
Shouldn't this be converted to `ButtonV2`?

Reply:
Yes, we can try doing that. I did not changed it since it was working fine.

Let me try ButtonV2 and then I will update you.